src/frontends/onnx/tests/CMakeLists.txt

```diff
@@ -154,3 +153,9 @@ install(TARGETS onnx_tests
 
 # process models
 add_dependencies(onnx_tests test_model_zoo)
+
+add_subdirectory(standalone_build)
+add_dependencies(onnx_tests onnx_fe_standalone_build_test)
+
+add_subdirectory(onnx_frontend_tests)
```

Comment:
What is the difference between all these tests? From my understanding it can be one binary, am I right?